modules/1_opencv/src/opencv_coins.cpp

```diff
@@ -1,14 +1,61 @@
 #include "opencv_coins.hpp"
 
+#include<iostream>
```

Comment:
please remove old task

Reply:
You need to start from master branch for every task:
```
git checkout master
```
```
git checkout -b new_task
```